src/faqs/benchmark.md

```diff
@@ -0,0 +1,21 @@
+# Benchmark
+
+`pgvecto.rs` has been focused on performance from the beginning. We have continuously monitored the performance of `pgvecto.rs` and compared it with other vector search libraries. The following benchmark results are from January 2024.
+
+The test is done on Google Cloud [n2-standard-8](https://cloud.google.com/compute/docs/general-purpose-machines#n2_series) (8 vCPUs, 32 GB RAM) with the `laion-768-5m-ip` dataset.
+
+## `pgvecto.rs` v.s. `pgvector`
+
+With the HNSW index, `pgvecto.rs` can achieve **2.5x** responses per second as `pgvector` can do with a slightly better precision of around 97%. This advantage increases when higher precision is required.
+
+![pgvecto.rs_vs_pgvector](./images/2024Jan_pgvectors_compare.png)
+
+With the filter enabled, `pgvecto.rs` can always reach more than **2x** responses per second as `pgvector` can do with different filter probabilities.
```

Comment:
`base` or `vbase` filter?

Reply:
Didn't find any info about that. @VoVAllen